### PR TITLE
chore: set weights for custom pallets in parachain environment

### PR DESCRIPTION
**Pull request by noisefloors**

### CODE REVIEW

**Comment by ETeissonniere** [comment]
@noisefloors what machine did you use to compute these? Does it match more or less the default specs we use on OnFinality?

**Comment by noisefloors** [comment]
@ETeissonniere This was computed on a machine matching the current hardware specifications of our Arcadia testnet, which should be very similar to our environment in OnFinality (and was the basis of previous Weight calculations).

**Comment by noisefloors** [comment]
I have Weight outputs for the system and runtime to add before we merge this in.

File: pallets/grants/src/weights.rs [hunk 1]
```diff
@@ -19,85 +19,44 @@
...
-		(50_695_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(7 as Weight))
-			.saturating_add(T::DbWeight::get().writes(4 as Weight))
-	}
-	fn cancel_all_vesting_schedules() -> Weight {
-		(128_611_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(8 as Weight))
-			.saturating_add(T::DbWeight::get().writes(6 as Weight))
-	}
-	fn overwrite_vesting_schedules() -> Weight {
```
Comment (aliXsed) [inline pallets/grants/src/weights.rs:74]: @noisefloors why we don't have these functions any more?

File: pallets/grants/src/weights.rs [hunk 2]
```diff
@@ -19,85 +19,44 @@
...
-		(40_916_000 as Weight)
-			.saturating_add(RocksDbWeight::get().reads(6 as Weight))
-			.saturating_add(RocksDbWeight::get().writes(5 as Weight))
-	}
-}
+impl WeightInfo for () {	fn add_vesting_schedule() -> Weight {
+		(103_704_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(9 as Weight))			.saturating_add(RocksDbWeight::get().writes(6 as Weight))	}	fn claim() -> Weight {
+		(62_420_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(8 as Weight))			.saturating_add(RocksDbWeight::get().writes(4 as Weight))	}	fn cancel_all_vesting_schedules() -> Weight {
+		(140_385_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(9 as Weight))			.saturating_add(RocksDbWeight::get().writes(6 as Weight))	}	fn overwrite_vesting_schedules() -> Weight {
+		(47_563_000 as Weight)			.saturating_add(RocksDbWeight::get().reads(7 as Weight))			.saturating_add(RocksDbWeight::get().writes(5 as Weight))	}}
```
Comment (aliXsed) [inline pallets/grants/src/weights.rs:62]: The formatting looks to be messed up

File: pallets/grants/src/weights.rs [hunk 3]
```diff
@@ -19,85 +19,44 @@
...
-		(40_916_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(6 as Weight))
-			.saturating_add(T::DbWeight::get().writes(5 as Weight))
-	}
-}
+impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {	fn add_vesting_schedule() -> Weight {
+		(103_704_000 as Weight)			.saturating_add(T::DbWeight::get().reads(9 as Weight))			.saturating_add(T::DbWeight::get().writes(6 as Weight))	}	fn claim() -> Weight {
+		(62_420_000 as Weight)			.saturating_add(T::DbWeight::get().reads(8 as Weight))			.saturating_add(T::DbWeight::get().writes(4 as Weight))	}	fn cancel_all_vesting_schedules() -> Weight {
+		(140_385_000 as Weight)			.saturating_add(T::DbWeight::get().reads(9 as Weight))			.saturating_add(T::DbWeight::get().writes(6 as Weight))	}	fn overwrite_vesting_schedules() -> Weight {
+		(47_563_000 as Weight)			.saturating_add(T::DbWeight::get().reads(7 as Weight))			.saturating_add(T::DbWeight::get().writes(5 as Weight))	}}
```
Comment (aliXsed) [inline pallets/grants/src/weights.rs:55]: Formatting issue

File: pallets/grants/src/weights.rs [hunk 4]
```diff
@@ -19,85 +19,44 @@
...
 use sp_std::marker::PhantomData;
 
 /// Weight functions needed for pallet_grants.
-pub trait WeightInfo {
-	fn add_vesting_schedule() -> Weight;
-	fn claim() -> Weight;
-	fn cancel_all_vesting_schedules() -> Weight;
-	fn overwrite_vesting_schedules() -> Weight;
-}
+pub trait WeightInfo {	fn add_vesting_schedule() -> Weight;	fn claim() -> Weight;	fn cancel_all_vesting_schedules() -> Weight;	fn overwrite_vesting_schedules() -> Weight;}
```
Comment (aliXsed) [inline pallets/grants/src/weights.rs:47]: Again formatting

**Comment by aliXsed** [comment]
@noisefloors would you please re-format your branch. I would also suggest that you add the weights for all the other system, governance and etc pallets to this PR.

**Comment by ETeissonniere** [comment]
@noisefloors @aliXsed can you make sure to recompute weights once #626 merges and merge this? Thanks!

File: pallets/grants/src/weights.rs (hunk 2 above)
Comment (ETeissonniere) [reply pallets/grants/src/weights.rs:62]: Could be an issue with the template file too. For now, run `cargo +nightly fmt` after a rebase

**Comment by simonsso** [comment]
I would say we should merge this asap and create a new PR based on latest and greatest when #626 is merged. We would deviate less from the master branch that way.

**Pull request closed without merging**